### ISSUE TITLE
java.lang.NoClassDefFoundError: net/citizensnpcs/trait/HologramTrait$HologramDirection

### ISSUE THREAD

**Issue by ChrissW-R1** [DenizenScript/Denizen-For-Bukkit#2557]
With the latest Citizens version (2.0.33 (BUILD #1 1.20.2)) we got the following issue in Denizen start:
```
[20:58:13] [Server thread/ERROR]: Error occurred while enabling Denizen v1.2.9-SNAPSHOT (build 1798-REL) (Is it up to date?)
java.lang.NoClassDefFoundError: net/citizensnpcs/trait/HologramTrait$HologramDirection
	at com.denizenscript.denizen.utilities.CommonRegistries.registerObjectTypes(CommonRegistries.java:268) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at com.denizenscript.denizen.utilities.CommonRegistries.registerMainObjects(CommonRegistries.java:88) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at com.denizenscript.denizen.Denizen.onEnable(Denizen.java:325) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[paper-1.20.2.jar:git-Paper-263]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.2.jar:git-Paper-263]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugin(CraftServer.java:644) ~[paper-1.20.2.jar:git-Paper-263]
	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugins(CraftServer.java:555) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:627) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:424) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1086) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[paper-1.20.2.jar:git-Paper-263]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
Caused by: java.lang.ClassNotFoundException: net.citizensnpcs.trait.HologramTrait$HologramDirection
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:197) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:164) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	... 15 more
[20:58:13] [Server thread/INFO]: [Denizen] Disabling Denizen v1.2.9-SNAPSHOT (build 1798-REL)
[20:58:13] [Server thread/ERROR]: Error occurred (in the plugin loader) while disabling Denizen v1.2.9-SNAPSHOT (build 1798-REL) (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "com.denizenscript.denizencore.flags.SavableMapFlagTracker.saveToFile(String)" because "com.denizenscript.denizencore.DenizenCore.serverFlagMap" is null
	at com.denizenscript.denizencore.DenizenCore.saveAll(DenizenCore.java:161) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at com.denizenscript.denizencore.DenizenCore.shutdown(DenizenCore.java:169) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at com.denizenscript.denizen.Denizen.onDisable(Denizen.java:477) ~[Denizen-1.2.9-b1798-REL.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:283) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.disablePlugin(PaperPluginInstanceManager.java:224) ~[paper-1.20.2.jar:git-Paper-263]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.disablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.2.jar:git-Paper-263]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:537) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:193) ~[paper-1.20.2.jar:git-Paper-263]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.2.jar:git-Paper-263]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugin(CraftServer.java:644) ~[paper-1.20.2.jar:git-Paper-263]
	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugins(CraftServer.java:555) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:627) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:424) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1086) ~[paper-1.20.2.jar:git-Paper-263]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[paper-1.20.2.jar:git-Paper-263]
	at java.lang.Thread.run(Thread.java:1623) ~[?:?]
```

**Comment by mergu** [comment]
Support is on discord @ https://discord.gg/Q6pZGSR
Feel free to open a forum post with your logs and stacktrace

**Comment by ChrissW-R1** [comment]
Should be solved with [1.2.9-b1799-REL](https://ci.citizensnpcs.co/job/Denizen/1799/)
https://github.com/DenizenScript/Denizen/commit/aabef89bb8378023ab222a5bff28b503d31b9c7e
